Updated SMS scheduling logic

ruhamyakmc opened the pull request:
Fixed SMS scheduling to use database follow-up dates and default appointment date for arm 8

ruhamyakmc committed:
Updated SMS scheduling logic

Fixed SMS scheduling to use database follow-up dates and default appointment date for arm 8

Werick reviewed (commented):
How do we handle cases where the two dates do not exist?
Looks like it will work only after updating two columns, this is not clear to me.

ruhamyakmc commented:
Good catch , will update the case and push an update..

ruhamyakmc committed:
Update baseline.xml

Corrected a typo and updated logic to skip video-related questions when participants are ineligible

ruhamyakmc committed:
eligibility check2 criteria( uganda )

Added two variables to eligibility check2 criteria( uganda )

Werick commented on AutomaticVariables.vb:
@ruhamyakmc  I think this is not correct, you want to add the countrycode/region checker make changes appropriately. Otherwise it will not work for Kenya.
For example you need to do something like this:
if countrycode = 1 then
added the Ugandan Criteria 
else
Kenyan Criteria as is
end if

Werick reviewed (commented):
@ruhamyakmc you can review my comment make updates then, It should be good to go.
Thank you
